modules/extensions-spring-support/src/test/resources/heisenberg-paged-operation-config.xml

```diff
@@ -53,16 +53,16 @@
     </flow>
 
     <flow name="consumeFailAtClosePagedOperation">
-        <heisenberg:fail-at-close-paged-operation failOn="#[payload]" config-ref="heisenberg"/>
+        <heisenberg:failing-connectivity-paged-operation failOn="#[payload]" config-ref="heisenberg"/>
```

Comment:
isn't this changing the test condition? You should leave these as they are and add new ones instead

Reply:
okay